en/micro-integrator/docs/references/connectors/redis-connector/redis-connector-example.md

```diff
@@ -0,0 +1,455 @@
+# Redis Connector Example
+
+The Redis connector allows you to access the Redis commands through the WSO2 EI. Redis is an open source (BSD licensed), in-memory data structure store, used as a database, cache and message broker. It supports data structures such as strings, hashes, lists, sets, sorted sets with range queries, bitmaps, hyperloglogs and geospatial indexes with radius queries.
+
+This connector enables developers to use an external Redis server as a cache or a database in the WSO2 EI mediation logic.
+
+## What you'll build
+
+Given below is a sample scenario that demonstrates how to work with the WSO2 EI Redis Connector and access the Redis server, using Redis commands.
+
+The user sends the request for invoke to an API to get WSO2 stock details. this REST call will get converted into a SOAP message and sent to the back-end service. While the response from the backend service is converted back to json and sent back to the API caller, WSO2 EI will extract stock volume details from the response and use **/getstockquote** operation to store it into configured Redis server.
+When users need to retrieve stock volumes collected, they can invoke `getstockvolumelength` operation. This example also demonstrates how users can manipulate this list by removing unwanted items from the list.
+
+> **Note**: In this scenario we need to set up the Redis Server in your local machine. Please refer the [Setting up the Redis Connector](redis-connector-configuration.md) documentation. Follow the steps listed under `Setting up the Redis server` section for setup the Redis server and `Set up the back-end service` section for setup the Stockquote service. 
+
+This example demonstrates how to use Redis connector to:
+
+1. Create a Redis list and add stock volume details to the Redis server. (In this example a Redis list is use to storing WSO2 stocks volume details. Since the “symbol” that is sent in the request is “WSO2”, the request is routed to the WSO2 endpoint. Once the response from the WSO2 endpoint is received, it is transformed according to the specified template and sent to the client. Then create a list and insert extracted details to the Redis list).
+2. Retrieve Redis list length from the created list.
+3. Retrieve list details and elements from the given range of list specified using Redis list length acquired above.
+4. Remove stock volume list items.
+
+All four operations are exposed via an `StockQuoteAPI` API. The API with the context `/stockquote` has four resources  
+
+* `/getstockquote/{symbol}` : Used to create a Redis list and add list items.
+* `/getstockvolumelength` : Retrieve information about the created list length.
```

Comment:
As a scenario, `/getstockvolumelength` is not much of use for the API caller. Shall we include it into `/getstockvolumelist` resource and do it as a service chain?

Reply:
updated.